tests/run-eval-tests.joke

```diff
@@ -37,7 +37,8 @@
       (println (str "Running test in subdirectory " test-dir)))
     (let [dir (str "tests/eval/" test-dir "/")
           filename "input.joke"
-          res (joker.os/sh-from dir exe filename) ; Someday add: :in "input.txt" (no :in support yet)
+          stdin (slurp-or (str dir "stdin.txt") :pipe)
+          res (joker.os/exec exe {:dir dir :args [filename] :stdin stdin}) ; Someday add: :in "input.txt" (no :in support yet)
```

Comment:
Looks like the comment is now obsolete and should be removed?

Reply:
Yes, since the functionality is there, but i wasn't sure how literal (or old) the comment was.

Reply:
> Looks like the comment is now obsolete and should be removed?

That was my comment. Idea there was to detect whether `input.txt` existed in the test's source directory and, if so, automatically set it to be on `stdin` via an `:in` (or now `:stdin`) keyword.

This would simplify at least one existing test, I think, so the test itself wouldn't need to open the file on startup.

IIRC, my intent was to have the file -- not a "slurp'ed" copy -- open on `stdin`. Looks like this new code might do the latter?

Also, I like the `stdin.txt` name better than `input.txt`, _especially_ if the file (not a slurp'ed copy) is open on `stdin`.

Reply:
Looks like slurp works fine for this use case and doesn't require additional support from `joker.os/exec`, so let's go with approach this for now.